content/rs/clusters/monitoring/prometheus-integration.md

```diff
@@ -26,8 +26,7 @@ You can use Prometheus and Grafana to:
 
 - Display Redis Enterprise Software metric data alongside data from other applications
 
-
-![grafana-prometheus](/images/rs/grafana-prometheus.png)
+{{<image filename="/images/rs/grafana-prometheus.png" alt="Graphic showing how Prometheus and Grafana collect and display data from a Redis Enterprise Cluster. Prometheus scrapes metrics from the Redis Enterprise cluster and Grafana queries those metrics for visualization.">}}{{< /image >}}
```

Comment:
Broken image. I'm not sure if the forward slash is the problem, but that's the main thing that stood out to me as different from the existing image shortcodes.
```suggestion
{{<image filename="images/rs/grafana-prometheus.png" alt="Graphic showing how Prometheus and Grafana collect and display data from a Redis Enterprise Cluster. Prometheus scrapes metrics from the Redis Enterprise cluster and Grafana queries those metrics for visualization.">}}{{< /image >}}
```